code/modules/reagents/chemistry/reagents/misc_reagents.dm

```diff
@@ -502,7 +503,10 @@
 			"You feel like telling a pun.")
 			to_chat(M, "<span class='warning'>[pick(clown_message)]</span>")
 		if(M.mind.assigned_role == "Mime")
-			update_flags |= M.adjustToxLoss(1.5 * REAGENTS_EFFECT_MULTIPLIER)
+			if(!(M.dna.species.reagent_tag & PROCESS_ORG)) // Only activates if the mob that is processing the reagent is Synthetic (IPC)
+				update_flags |= M.adjustFireLoss(1.5 * REAGENTS_EFFECT_MULTIPLIER, robotic = TRUE)  // The ", robotic = TRUE" just means it affects robotic limbs, which an IPC is fully robotic
+			else // Ensures the mob that is processing the reagent is Organic to apply toxin damage // Using burn instead of toxin for IPCs since IPCs don't suffer toxin and poison can feel similar to burns
```

Comment:
```suggestion
				update_flags |= M.adjustFireLoss(1.5 * REAGENTS_EFFECT_MULTIPLIER, robotic = TRUE) // Deal burn instead of toxin for IPCs, since IPCs don't suffer toxin and poison can feel similar to burns
			else // Ensures the mob that is processing the reagent is Organic to apply toxin damage
```

feel free to put the removed comment back. just dont overly see it being to needed. however the comment explaining the burn damage should be on the respective line